code/modules/clothing/clothing.dm

```diff
@@ -282,6 +282,7 @@
 * SEE_TURFS // can see all turfs (and areas), no matter what
 * SEE_PIXELS// if an object is located on an unlit area, but some of its pixels are
 *           // in a lit area (via pixel_x,y or smooth movement), can see those pixels
+* SEE_SCIENCE // can see science points
```

Comment:
That's not how this works at all.
These are  BYOND defines you cannot extend this functionality.
https://www.byond.com/docs/ref/#/mob/var/sight

Reply:
oh that's leftover.